data/dictionary/amcon.yaml

```diff
@@ -1,4 +1,4 @@
-word: Amcon
+word: Am Con
```

Comment:
Was this already capitalized and I misread it? Looks like you can ignore this comment.